src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx

```diff
@@ -60,11 +60,11 @@ const cardObjects = [
                 <FontAwesomeIcon icon="sitemap" color="#0081a1" size="lg" />
             </div>
         ),
-        headline: 'Data Model',
-        text: 'Learn how our data is organized',
+        headline: 'About the Data',
+        text: 'Read important data disclosures',
         buttonText: (
             <>
-                View the model&nbsp;&nbsp;
+                View the disclosures;
                 <FontAwesomeIcon icon="arrow-right" />
             </>
         ),
```

Comment:
at line 71, the link needs to change so this click will open the new About the Data module

Reply:
fixed

Reply:
trying to find the correct link now

Reply:
just to make sure is the the correct link
https://www.usaspending.gov/data-sources

Reply:
oh man... I knew that.. so dumb..
Ill see if I can link it